scripts/install

```diff
@@ -93,7 +93,7 @@ ynh_setup_source "$final_path"
 # INSTALL SEARX IN A VIRTUALENV
 #=================================================
 
-virtualenv --system-site-packages "$final_path"
+virtualenv --python=/usr/bin/python3 --system-site-packages "$final_path"
 set +u; source $final_path/bin/activate; set -u
 pip install -U setuptools
```

Comment:
Use `pip3` instead of `pip`

Reply:
Oh yes. I'm not really used to debian defaults… You're right it totally make sense to do it that way.

Reply:
Oh, sorry in fact you don't have to use pip3 (your comment was mis-aligned in the last window, thus I thought you were speaking about the virtualenv line). As we initiated virtualenv with python3, pip in that virtualenv is in fact pip3.

Reply:
I was not sure. Thank you for your explanation!